examples/allocation/zig/README.md

```diff
@@ -0,0 +1,16 @@
+## Zig allocation example
+
+This example shows how to pass strings in and out of a Wasm function defined in Zig, built with `zig build`.
+
+Ex.
+```bash
+$ go run greet.go wazero
+wasm >> Hello, wazero!
+go >> Hello, wazero!
+```
+
+Under the covers, [greet.zig](testdata/greet.zig) does a few things of interest:
+* Uses `@ptrToInt` to change a Zig pointer to a numeric type.
+* Uses `@intToPtr` to build back a string from a pointer, len pair.
```

Comment:
This seems a stale comment

Reply:
Updated. `@ptrToInt` is still used.
https://github.com/tetratelabs/wazero/pull/741/commits/298914b368773bb36a0475957e2544285652f74f